libs/MeshKernel/include/MeshKernel/CurvilinearGrid/CurvilinearGrid.hpp

```diff
@@ -40,10 +40,16 @@
 namespace meshkernel
 {
     /// @brief A class representing a curvilinear grid
-    class CurvilinearGrid final : public Mesh
+    class CurvilinearGrid final
     {
 
     public:
+        /// @brief Typedef for edge indices
+        using CurvilinearEdgeNodeIndices = std::array<CurvilinearGridNodeIndices, 2>;
+
```

Comment:
Is it possible to use the Edge type, defined in Entities.hpp

Reply:
I think it is possible. But it requires a few other changes in other parts that deserve a separate pr